pandas/core/resample.py

```diff
@@ -365,8 +365,9 @@ def _apply_loffset(self, result):
             the result of resample
         """
 
+        loffset_types = (DateOffset, timedelta, np.timedelta64)
         needs_offset = (
-            isinstance(self.loffset, (DateOffset, timedelta)) and
```

Comment:
go ahead and inlinethese (add an overall comment if you would)

cc @jbrockmendel we should define scalar function helpers for this as we repeat lots of this in ops....